sql/branch.sql

```diff
@@ -0,0 +1,3 @@
+BEGIN;
+    UPDATE payment_instructions SET due = 0 WHERE due > 9.41;
```

Comment:
The five includes [these four](https://github.com/gratipay/gratipay.com/pull/3980#issuecomment-206063806).